index.html

```diff
@@ -0,0 +1,178 @@
+                        <li class="nav__item"><a href="#work" class="nav__link">My Work</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <head>
+        
+<!--Introduction -->
+        <section class="intro" id="home">
+            <h1 class="section__title section__title--intro">
+            Welcome to <strong>Karla's Website</strong>
+            </h1>
+            <p class="section__subtitle section__subtitle--intro">Hello | Hola | Ciao | Bonjour | こんにちは | നമസ്കാരം | नमस्ते  </p>
+            <img src="img/About me Photo.JPG" alt="a picture of me" class="intro__img">
+            <div id="menu-outer">
+                <div class="table">
+                    <ul id="horizontal-list">
+                        <li class="nav-link"><a href="about.html">My About Page</a></li>
+                        <li class="favorite-link"><a href="portfolio.html">My Portfolio of Projects</a></li>
+                        <li class="photo-link"><a href="blog.html">My Photo Blog</a></li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+        </head>
+
+<!-- My services  -->
+        <section class="my-services" id="services">
+            <h2 class="section__title section__title--services"> What I do</h2>
+            <div class="services"><!--Helps create the three columns -->
+                <div class="service">
+                    <h3>Design & Development</h3>
+                    <p>Design and create customizable buttons.</p>
+                </div> <!-- / service  -->
+
+                <div class="service">
+                    <h3>Web Page Enhancements</h3>
+                    <p>Work with you until we find your </p>
+                </div> <!-- / service -->
+
+                <div class="service">
+                    <h3>Code Sparing for Prep Interviews</h3>
+                    <p>Focused primarely on Python and Javascript we round up and prep for interviews</p>
+                </div> <!-- / service -->
+            </div> <!-- / services -->
+
+            <a href="#work" class="btn">My Work</a>
+        </section>
+
+<!--About me-->
+        <section class="about-me" id="about">
+            <h2 class="section__title section__title--about">Who I am</h2>
+            <p class="section__subtitle section__subtitle--about"> Photographer | Adventure Seeker | Nurturer | Health Fanatic | Computer Engineering Student</p>
+            <!-- <style>
+                body{
+                    background-image: url("img/Beach_waves_Shore.JPG");
+                }
+            </style> -->
+            <div class="about-me__body">
+                <p>I am a student of computer science. Diving into this field with high spirits and full of life!</p>
+            </div>
+
+            <img src="img/Everyday_Shots_Buttercup_inSunset.JPG" class="about-me__img">
```

Comment:
Images should have an `alt` attribute for accessibility and in case the image doesn't render.